hack/local-up-cluster.sh

```diff
@@ -30,6 +30,7 @@ RUNTIME_CONFIG=${RUNTIME_CONFIG:-""}
 KUBELET_AUTHORIZATION_WEBHOOK=${KUBELET_AUTHORIZATION_WEBHOOK:-""}
 KUBELET_AUTHENTICATION_WEBHOOK=${KUBELET_AUTHENTICATION_WEBHOOK:-""}
 POD_MANIFEST_PATH=${POD_MANIFEST_PATH:-"/var/run/kubernetes/static-pods"}
+KUBELET_IMAGE={KUBELET_IMAGE:"k8s.gcr.io/hyperkube-amd64:-latest"}
```

Comment:
This should be `KUBELET_IMAGE={KUBELET_IMAGE:-"k8s.gcr.io/hyperkube-amd64:-latest"}`

Reply:
i thought i fixed this earlier .. looks like i forgot to push it up.

Reply:
Done. thanks @cblecker